packages/recomponents/src/components/r-popper/r-popper.vue

```diff
@@ -194,8 +194,10 @@
                 this.isPopperVisible = visible;
                 await this.$nextTick(); // await for contentEl to be visible before trying to access it
                 if (this.contentEl && this.contentEl.hasChildNodes()) {
-                    this.contentEl.firstChild.style = `transition-duration: ${this.duration}s`;
-                    this.positionContent();
+                    if (this.contentEl.firstChild.classList) {
+                        this.contentEl.firstChild.style.transitionDuration = `${this.duration}s`;
```

Comment:
fixed the position issue.
just update transition-duration here, don't override all the style, I checked a couple of hours and found why !!!